export/main.ts

```diff
@@ -99,19 +106,27 @@ const input = [];
 for(const proj of projects) {
     input.push(limit(async () => {
         const data = await fetchProjectData(proj.id, proj.group_name)
+        let msg = "";
+
+        if(data) {
+            msg = data.name ? data.name.padEnd(50, ' ') : "".padEnd(50, ' ');
+            writeProject(proj.id, data);
+        } else {
+            msg = `Project '${proj.id}' has no job runs`;
+        }
+
         completed++;
 
         bars.render([
            {
              completed: completed,
              total: projects.length,
-             text: data.name ? data.name.padEnd(50, ' ') : "",
+             text: msg,
```

Comment:
Should the `padEnd` move back to this line? I'm not sure what it's for, but right now we apply it to the ends of both expressions in an if/else.

Reply:
Yes, probably. It's just for formatting during processing. I just moved it here.